src/VolumeCache.ts

```diff
@@ -21,7 +21,7 @@ type CacheEntry = {
   // TODO allow more types of `TypedArray` to be stored together in the cache?
   data: Uint8Array;
   /** The subset of the volume covered by this entry */
-  extent: CacheEntryExtent;
+  region: Box3;
```

Comment:
Region vs. subregion here? Does subregion make more sense to be consistent with its usage elsewhere?